components/atom/spinner/src/SUILoader/index.scss

```diff
@@ -0,0 +1,45 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-accent-atom-spinner-loader: color-variation($c-accent, 1) !default;
+$c-primary-atom-spinner-loader: color-variation($c-primary, -1) !default;
+$sz-atom-spinner-oval: $sz-icon-s;
+$sz-atom-spinner: $sz-atom-spinner-oval * 2;
+
+@keyframes sui-atom-spinner {
```

Comment:
For readability, perhaps you could scope the variables here.

```sass
$sz-oval: $sz-atom-spinner-oval;
$c-accent: $c-accent-atom-spinner-loader;

/* etc */
```

Reply:
Looks like it's not possible to define vars inside keyframes

Reply:
😞